db/migrate/20190624131249_comments_have_postid.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsHavePostid < ActiveRecord::Migration[5.2]
+  def up
+    execute <<-SQL
+      DELETE FROM comments;
+    SQL
```

Comment:
Does this delete all comments? Why do that?

Reply:
I'm adding a required field for parent_post_id, which makes any old comments unloadable without that field :/

Reply:
Ah, got it. That makes sense. 👍